ndcube/tests/test_ndcube.py

```diff
@@ -827,3 +827,9 @@ def test_reproject_exact_incompatible_wcs(ndcube_4d_ln_l_t_lt, wcs_4d_lt_t_l_ln,
     with pytest.raises(ValueError):
         _ = ndcube_4d_ln_l_t_lt.reproject_to(wcs_4d_lt_t_l_ln, algorithm='exact',
                                              shape_out=(5, 10, 12, 8))
+
+
+def test_plot_docstring():
+    cube = NDCube([], astropy.wcs.WCS())
+
+    assert cube.plot.__doc__ == cube.plotter.plot.__doc__
```

Comment:
You could use this: https://docs.python.org/3/library/inspect.html#inspect.signature to compare the signature as well?

Reply:
Ah yes, good call.

Reply:
So actually it turns out the signatures aren't the same anyway, so this won't work right now. From what I remember of figuring out how to do this in the first place I think the signature was slightly more fiddly to copy accross, and I'm not sure how much difference it makes to the user if the docstring is there. So I guess it's up to @DanRyanIrish whether we want to incur a short delay while I fix that or merge it as it and open a new issue for the signature to come back to.

Reply:
Some of the signature did get copied:
```
In [8]: inspect.signature(ds.plotter.plot)
Out[8]: <Signature (axes=None, plot_axes=None, axes_coordinates=None, axes_units=None, data_unit=None, wcs=None, **kwargs)>

In [9]: inspect.signature(ds.plot)
Out[9]: <Signature (plot_axes=None, axes_coordinates=None, axes_units=None, data_unit=None, wcs=None, **kwargs)>
```
Not entirely sure where the bonus `axes=` comes from though.